templates/paragraphs/paragraph--contact-card.html.twig

```diff
@@ -0,0 +1,52 @@
+{%
+  set classes = [
+    'contact-card',
+    view_mode ? 'contact-card---' ~ view_mode|clean_class,
+    content.field_contact_image|render ? 'contact-card--with-image'
+  ]
+%}
+
+
+{% block paragraph %}
+  <div{{ attributes.addClass(classes) }}>
+    {% if content.field_contact_image|render %}
+      <div class="contact-card--image-container">
+        {{ content.field_contact_image }}
+      </div>
+    {% else %}
+      <div class="contact-card--color-line"></div>
+    {% endif %}
+    <div class="contact-card--information-container">
+    {% if content.field_contact_name|render %}
+      <h3 class="contact-card--person-information contact-card--person-name">
```

Comment:
How can we know that there's a h2 before this?

Reply:
I was going to say that the paragraph has a h2 as its title but since it's not mandatory, that is a very good question

Reply:
Yep, maybe a condition based on the h2 title, either h2 or h3 depending on if the title is visible or not.